www/src/app/reducers/updates.js

```diff
@@ -0,0 +1,25 @@
+import { types } from '../actions';
+
+const defaultState = {
+  inbox: [],
+  done: [],
+  goal: []
```

Comment:
if we choose to store updates in multiple arrays, one for each possible status, the code in `InboxContainer.handle*` methods will get a bit more complex.  For each change made by the user, we'll need to remove the update from the array corresponding to its previous status and push into the array corresponding to the new status.

Instead, can rather we keep all updates in a single `updates` array, independent of their status?  This makes it easier to change the status and I think it gives us more flexibility right now.

I'm not opposed to changing the layout of the data store, but perhaps it can be done in another PR.